core/eolearn/core/utils/parsing.py

```diff
@@ -81,25 +98,35 @@ class FeatureParser:
     - For `get_renamed_features` a list of triples `(feature_type, old_name, new_name)`.
     """
 
-    def __init__(self, features: Union[dict, Sequence], allowed_feature_types: Optional[Iterable[FeatureType]] = None):
+    def __init__(
+        self,
+        features: Union[SingleFeatureSpec, MultiFeatureSpec],
```

Comment:
`Union[SingleFeatureSpec, MultiFeatureSpec]` is used quite a few times throughout the MR. Would it make sense to add a new type for this, such as `AnyFeatureSpec` or something, or is that just complicating?

Reply:
it would make sense, a lot of the tasks will use such a notation.

Reply:
I'll resolve this thread here, let's continue in https://github.com/sentinel-hub/eo-learn/pull/469#discussion_r978494404